tests/test_timeseries.py

```diff
@@ -14,6 +15,15 @@ def test_fill_series():
     assert fill_series(y, 2006) == 3.
 
 
+def test_fill_series_years():
```

Comment:
```suggestion
def test_fill_series_datetime():
```
and suggest to rename the function above to `test_fill_series_year()`